drive_controller:
    type        : "diff_drive_controller/DiffDriveController"
    left_wheel  : 'left_wheel_hinge'
    right_wheel : 'right_wheel_hinge'
    publish_rate: 100
    
    enable_odom_tf: false
    
    pose_covariance_diagonal : [0.00001, 0.00001, 1000000000000.0, 1000000000000.0, 1000000000000.0, 0.001]
    twist_covariance_diagonal: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]

    # Wheel separation and radius multipliers
    wheel_separation_multiplier: 1.0 # default: 1.0
    wheel_radius_multiplier    : 1.0 # default: 1.0

    # Base frame_id
    base_frame_id: base_link

    linear:
      x:
        has_velocity_limits    : true
        has_acceleration_limits: true
        max_acceleration       : 4.0 
        min_acceleration       : -6.0
        max_velocity           : 0.7
    angular:
      z:
        has_velocity_limits    : true
        has_acceleration_limits: true
        max_acceleration       : 20.0
        min_acceleration       : -30.0
        max_velocity           : 10.0
